transcom/mymove
# MB-16193 Create a swagger endpoint for creating a service request document upload

## [MB-16193](https://dp3.atlassian.net/browse/MB-16193)

## Summary

Add new swagger endpoint for creating a service request document upload.
I pretty much followed the existing format for what the swagger definition looked like for uploading a proof of service document (createUpload).

### Open Questions
Am I supposed to be changing two files here? 

Is there anything else I need to do? 
Creating a handler and service object will be handled by the following tickets.
https://dp3.atlassian.net/browse/MB-16192
https://dp3.atlassian.net/browse/MB-16222

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Run `make server_run`. 
2. Everything should still operate as expected

Additionally, you should be able to visit this page and see documentation for a new endpoint.
https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/transcom/mymove/yan-MB-16193-create-a-swagger-def-for-uploading-service-request-documents/swagger/prime.yaml#tag/mtoServiceItem/operation/createServiceRequestDocumentUpload


### Backend




[MB-16193]: https://dp3.atlassian.net/browse/MB-16193?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
